lessons/Algebra2/exploring-logarithmic-relationships/langs/en-us/index.adoc

```diff
@@ -0,0 +1,163 @@
+
+Imagine that the scatter plot is printed on a sheet of rubber, and can be stretched or squished in any way we want. Data Scientists often use *transformations* to stretch their data into shapes that are easier to use, and then reverse the transformation when they are done.
+
+[.lesson-instruction]
+- Complete questions 1-6 in @printable-exercise{transforming-wealth.adoc}
+- Discuss as a class: what does `build-column` do?
+
+=== Investigate
+
+Since the values on the y-axis (`median-lifespan`) appear grow logarithmically, we could _transform_ the x-axis (`pc-gdp`) to grow logarithmically as well. With both x- and y-valus growing logarithmically, they would appear grow at a constant rate relative to one another. A constant rate will appear linear, which allows us to use `lr-plot` to find the optimal model!
+
+How can we transform this plot so that the `pc-gdp` column is placed on a logarithmic scale?
+
+[.lesson-instruction]
+Complete questions 7-8 on @printable-exercise{transforming-wealth.adoc}
+
+@right{@image{images/wealth-v-life-transformed-linear.png, 550}} Success! After transforming the `pc-gdp` column to `log-pc-gdp`, a scatter plot shows a more linear pattern in the data! Pyret's `lr-plot` tool computes the best possible linear model for our transformed data, determining it to have a slope of `5.169` and a y-intercept of `24.264`. Our @math{R^2} has jumped to 0.663, showing a vastly better correlation than before.
+
+@clear
+
+With the transformation applied, our linear model (in both function and Pyret notation) is:
+
+- @math{f(x) = 5.169x + 24.264}
+- @show{(code '(define (f x) (+ (* 5.169 x) 24.264)))}
+
+At each point in our linear model, @math{f} is the _predicted median lifespan_, and @math{x} is the _log of per-capita gdp_. But we want is for @math{x} to represent the **original, un-transformed** value, simply using _per-capita gdp_ as-is.
+
+Instead of transforming our dataset and then fitting a linear model, we want to transform the model itself and then fit it to the original dataset. We already know the translation requires taking the log of `per-capita-gdp`, so we can simply compose our linear function with a log function:
+
+- @math{g(x) = f(log(x))}
+- @show{(code '(define (g x) (f (log x))))}
+
+Of course, we could also just define this composition as a new function altogether, replacing @math{x} in our linear model with @math{log(x)}:
+
+- @math{h(x) = 5.169 \times log(x) + 24.264}
+- @show{(code '(define (h x) (+ (* 5.169 (log x)) 24.264)))}
+
+[.lesson-instruction]
+Complete the remainder of in @pages{transforming-wealth.adoc}
+
+The resulting logarithmic model can be fit to our original scatter plot, showing a much better fit than our 2-point-derived estimates.
+
+@center{@image{images/wealth-v-life-log.png}}
+
```

Comment:
Wondering if there needs to be another worksheet for students to practice this process on their own?

Reply:
I have just the thing in mind! Pushed.